index.ts

```diff
@@ -1,6 +1,6 @@
 export * from "./lib/types"
 
-export { JSONToXD } from "./lib/JSONtoXD"
+export { JSONToXD, getXDClueAnswer } from "./lib/JSONtoXD"
```

Comment:
I'm not sure on the naming of this function, what is it actually trying to do? 

- `resolveFullClueAnswer`  maybe?